repos/system_upgrade/el9toel10/models/firewalldglobalconfig.py

```diff
@@ -0,0 +1,31 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class FirewalldGlobalConfig(Model):
+    """
+    The model contains firewalld global configuration. The configuration is
+    usually located at /etc/firewalld/firewalld.conf.
+    """
+    topic = SystemInfoTopic
+
+    # Defaults for RHEL-10.
+    #
+    defaultzone = fields.String(default='public')
+    cleanuponexit = fields.Boolean(default=True)
+    cleanupmodulesonexit = fields.Boolean(default=False)
+    ipv6_rpfilter = fields.String(default='yes')
+    individualcalls = fields.Boolean(default=False)
+    logdenied = fields.String(default='off')
+    firewallbackend = fields.String(default='nftables')
+    flushallonreload = fields.Boolean(default=True)
+    reloadpolicy = fields.String(default='INPUT:DROP,FORWARD:DROP,OUTPUT:DROP')
+    rfc3964_ipv4 = fields.Boolean(default=True)
+    nftablesflowtable = fields.String(default='off')
+    nftablescounters = fields.Boolean(default=False)
+    nftablestableowner = fields.Boolean(default=False)
+
+    # These have been removed in RHEL-10.
+    #
+    allowzonedrifting = fields.Boolean(default=False)
```

Comment:
we will go through this PR most likely after we finish work on CTC1 builds (~2w). however, `allowzonedrifting` has been already removed in rhel9 - at least based on existing actors for IPU 8 -> 9.

Reply:
It's not removed. Just deprecated and ignored. I'll change the comment.